### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MHDGhanemBalhawan**

### CODE REVIEW

**Review by fentonfentonfenton — COMMENTED** [review]
I was able to install Postgres and insert into the db from the Postgres CLI:
image

Then when I visit the `all-events` endpoint in the backend I see my data!:
image

**TEST PASSED** 🆗 

I would like to see the instructions made more user friendly. For instance, here is the terminal history from when I installed the db on OSX:

```
brew install postgresql
brew services start postgresql
psql
```
We enter the Postgres CLI now:
```


create database events_calendar;
create user eventsuser;
alter user eventsuser with encrypted password 'eventsuser147';
grant all privileges on database events_calendar to eventsuser;

```
then i quit out with cntrl+d and then went back in as the new user:
```
psql events_calendar -h localhost -U eventsuser
 ```
```
CREATE TABLE events_tbl (
id serial primary key,
lesson VARCHAR(55),
event_date Date,
description text
);
INSERT INTO events_tbl (lesson, event_date, description) values ('html-1',
       '11-09-2019', 'Jack is really cool.');
```


I don't know whether this was the right order or if each step was necessary and it might help if there is a step by step instruction list for the next person to use?

Great work for getting it working !

**Comment by dahfool** [comment]
@jfbookingbug If you are happy with the changes can you please approve.

**Comment by dahfool** [comment]
@MHDGhanemBalhawan Is this PR ready to be reviewed ?

**Review by fentonfentonfenton — CHANGES_REQUESTED** [review]
Please se my earlier comment and make sure the README file is a step by step tutorial in the correct order with no additional / unused steps [you might find it easier to start again and go through it yourself].

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+  config /etc/postgresql/9.5/main
+  data   /var/lib/postgresql/9.5/main
+  locale en_GB.UTF-8
+  socket /var/run/postgresql
+  port   5432
+
+User: postgres
+-----------------------------
+
+connect to the sever:
```
Comment (dahfool) [inline README.md]: Should say connect to `connect to database server`

File: README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+User: postgres
+-----------------------------
+
+connect to the sever: 
+
+sudo -u postgres psql
+
+-----------------------------
+
+Create databse:
```
Comment (dahfool) [inline README.md]: Create `database`

File: README.md (hunk 1 above)
Comment (MHDGhanemBalhawan) [reply README.md]: I fixed it.

File: README.md (hunk 2 above)
Comment (MHDGhanemBalhawan) [reply README.md]: I fixed it, too.

File: README.md [hunk 3]
```diff
@@ -1 +1,67 @@
...
+-----------------------------
+
+Insert data into the table
+
+INSERT INTO events_tbl (lesson, event_date, description) values ('html-1',
+       '01-09-2018', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.');
+
+-----------------------------
+
+export database to home directory:
```
Comment (fentonfentonfenton) [inline README.md:62]: why will we need to do this? add the why to the readme

File: README.md [hunk 4]
```diff
@@ -1 +1,67 @@
 # Events calender
+
+
```
Comment (fentonfentonfenton) [inline README.md:3]: Mention these instructions are for a ubuntu/debian distro

File: README.md [hunk 5]
```diff
@@ -1 +1,67 @@
...
+Install postgres:
+apt-get install postgresql
+
+-----------------------------
+Data directory on Ubuntu
```
Comment (fentonfentonfenton) [inline README.md:8]: i don't understand the relevance of this block, do we need it?

**Review by fentonfentonfenton — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1 +1,67 @@
 # Events calender
```
Comment (dahfool) [inline README.md:1]: `calendar` not calender

**Comment by dahfool** [comment]
@MHDGhanemBalhawan If you use Markdown you can format your readme. check this out https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

File: README.md (hunk 3 above)
Comment (MHDGhanemBalhawan) [reply README.md:62]: OK I'll remove this line.

File: README.md (hunk 4 above)
Comment (MHDGhanemBalhawan) [reply README.md:3]: I'll mention that.

File: README.md (hunk 5 above)
Comment (MHDGhanemBalhawan) [reply README.md:8]: This was to show if the installation is success they will see this message.

File: README.md (hunk 6 above)
Comment (MHDGhanemBalhawan) [reply README.md:1]: OK.

**Pull request closed without merging**